### DISCUSSION TITLE
Add method for acquiring bot display name

### DISCUSSION THREAD

**Discussion by Zer0-One** [TwitchIO/TwitchIO#412] [Questions] [answered]
I'm trying to identify when someone mentions the bot (e.g `@BotNameHere`). Please consider either adding a Client property that contains the bot's display name, or adding a Message property that describes if the message is a mention to the bot.

**Comment by chillymosh** [comment]
This isn't an issue really and could have been a discussion.
self.nick / self.bot.nick already exists so you can simply use this to check in your message.content.lower()

**Comment by Zer0-One** [comment]
Is there ever a situation where the IRC nick differs from a lowercased displayname or username? If not, then maybe the naming here could be made slightly less confusing.

\>could have been a discussion

I don't use Discord.

**Comment by chillymosh** [comment, marked as answer]
A display name is simply the username / login (which is always lowercase) with custom capitalisation.

**Comment by chillymosh** [comment]
@Zer0-One please see above comment. Just to add to it, the reason it is called `nick` is because that is what you send when connecting to IRC. 
It was also a required kwarg.
